apps/cms/lib/page/event.ex

```diff
@@ -96,9 +106,38 @@ defmodule CMS.Page.Event do
     }
   end
 
-  @spec past?(t, Date.t()) :: boolean
-  def past?(event, now) do
-    Date.compare(event.start_time, now) == :lt
+  @spec started_status(
+          NaiveDateTime.t() | DateTime.t(),
+          NaiveDateTime.t() | DateTime.t() | nil
+        ) :: status
+  # Events have DateTime start/ends.  Teasers have NaiveDateTimes.
+
+  def started_status(%NaiveDateTime{} = start, stop) do
+    started_status(
+      convert_using_timezone(start, ""),
+      if(is_nil(stop), do: nil, else: convert_using_timezone(stop, ""))
+    )
+  end
+
+  def started_status(start, nil) do
+    cond do
+      time_is_greater_or_equal?(now(), start) and Date.compare(now(), start) === :gt -> :ended
```

Comment:
Since `time_is_greater_or_equal` internally does a `DateTime.compare`, would it make more sense to only look for `DateTime.compare(now(), start) === :gt` in this particular case?

Reply:
Love all these tips!  Thanks!